src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -79,7 +79,7 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
   // url param
   courseId: string = '';
   feedbackSessionName: string = '';
-  isEditingMode: boolean = false;
+  isEditingMode: boolean = true;
```

Comment:
Is this neccessary?

Reply:
It is not necessary for the change to work. I changed it because I thought it would reflect the state better since the page will open in the editing mode after routing. On second thought, I will remove it.